pandas/tests/groupby/test_grouping.py

```diff
@@ -511,6 +511,18 @@ def test_groupby_level_index_names(self):
         with pytest.raises(ValueError, match=msg):
             df.groupby(level="foo")
 
+    def test_groupby_level_columns_names(self):
```

Comment:
is there a sufficient test with axis=0 (otherwise parameterize this one)

Reply:
will do, thanks